frontend/src/reviews/ReviewComments.tsx

```diff
@@ -31,23 +46,92 @@ export default function ReviewComments({ release, accessRequest }: ReviewComment
     ...semverOrAccessRequestIdObject,
   })
 
-  const reviewDetails = useMemo(
-    () =>
-      reviews.map((review) =>
-        review.responses.map((response) => <>{response.decision && <ReviewDecision response={response} />}</>),
-      ),
-    [reviews],
-  )
+  const reviewDetails = useMemo(() => {
+    let decisionsAndComments: Array<ReviewResponseKind> = []
+    reviews.forEach((review) => {
+      review.responses.forEach((response) => decisionsAndComments.push(response))
+    })
+    if (release && release.comments) {
```

Comment:
Is the check for `&& release.comments` needed anymore? Looks like it's been updated so that the array is always there now